engine/render/proto/render/font_ddf.proto

```diff
@@ -56,8 +56,8 @@ message GlyphBank
         optional uint32 width             = 2 [default = 0];    // glyph width
         optional float  advance           = 3 [default = 0.0];
         optional float  left_bearing      = 4 [default = 0.0];
-        optional uint32 ascent            = 5 [default = 0]; // WHY 32 bits???
-        optional uint32 descent           = 6 [default = 0]; // WHY 32 bits???
+        optional int32 ascent             = 5 [default = 0]; // WHY 32 bits???
+        optional int32 descent            = 6 [default = 0]; // WHY 32 bits???
```

Comment:
While we use the baseline as the origin, and the values as "distance from baseline", it's possible that the values are negative. This can happen when the entire glyphs is fully on only one side of the base line.
E.g. `_` can have a negative ascent, as it's below the base line.